packages/example-app-base/src/common/_common.scss

```diff
@@ -4,3 +4,6 @@
 @import './focusBorder';
 @import './semanticColorVariables';
 
+// UHF sizing
```

Comment:
Same comment as above--would be good to expand on this comment a little. Calling out that they're meant for the Fabric website would be useful.